<?xml version="1.0"?>
<launch>

  <arg name="map_name" default="icra" />

  <!--<arg name="map_path" default="/home/ubuntu/thundershot/maps/map.yaml" /> -->

  <node pkg="particle_filter" exec="pf_node" name="particle_filter" output="screen" launch-prefix="bash -e valgrind --leak-check=full --track-origins=yes" >
    <!-- sub topics -->
    <remap from="/map" to="/map"/>
    <remap from="/initialpose" to="/initialpose"/>
    <remap from="/odom" to="/ego_racecar/odom"/>
    <remap from="/scan" to="/scan"/>

    <!-- pub topics -->
    <remap from="/thundershot/pf/position" to="/thundershot/pf/position"/>
    <remap from="/thundershot/pf/particles" to="/thundershot/pf/particles"/>

    <!-- configurations -->
    <param name="conf_path" value="$(find-pkg-share particle_filter)/conf/particle.yaml"/>
    <param name="ekf_conf_path" value="$(find-pkg-share particle_filter)/conf/ekf.yaml"/>
  </node>

  <!-- Launch pure pursuit
  <group>
    <include file="$(find-pkg-share purepursuit)/launch/purepursuit_h2h.launch.xml" />
  </group> -->

  <!-- Launch frenet planner
  <group>
    <include file="$(find-pkg-share frenet_planner)/launch/frenet_planner.launch.py" />
  </group> -->

  <!-- Launch path publisher
  <group>
    <include file="$(find-pkg-share path_publisher)/launch/path_publisher_tshot.launch.xml">
        <arg name="map_name" value="$(var map_name)" />
    </include>
  </group> -->
	
  <!-- Launch map server
  <node pkg="nav2_map_server" exec="map_server" name="map_server" >
    <param name="yaml_filename" value="$(find-pkg-share maps)/maps/$(var map_name)/map.yaml" />
  </node>

  <node pkg="nav2_util" exec="lifecycle_bringup" name="lifecycle_bringup" args="map_server"/> -->

</launch>
